src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
-
+        Engine engine = new Engine("JZ-1", 600, 550, 9.5, 7);
+        Engine engine2 = new Engine("JZ-2", 700, 550, 12.5, 6);
```

Comment:
please remove `engine2` variable and setting it to `clonedToyota`

try to read the task description again please